### PR TITLE
Updated dev install docs

### PR DESCRIPTION
**Pull request by jmcameron**

### CODE REVIEW

**Review by jniles — APPROVED** [review]
LGTM

**Comment by jmcameron** [comment]
@jniles and @mbayopanda , please double-check my updates to RELEASE.md

File: RELEASES.md [hunk 1]
```diff
@@ -4,26 +4,27 @@ RELEASES
...
+13. Verify the release was created and that the binary <version>.tar.gz file
     is in the assets for the release.
 
 
 (*) Optional, Best Practice: Test the latest database changes on a production database.
 
 1. Change your `.env` so the `$DB_NAME` variable is the correct one for a production database.
 2. Create your database migration script by running `yarn migrate`.  This will create a file `migration-$DB_NAME.sql`.
-3. Catenate the latest release file into the output `cat server/models/migrations/next/migrate.sql >> migration-$DB_NAME.sql`.
+3. Concatenate the latest release file into the output `cat server/models/migrations/next/migrate.sql >> migration-$DB_NAME.sql`.
```
Comment (jniles) [inline RELEASES.md]: Catenate is the proper term 😆

File: RELEASES.md [hunk 2]
```diff
@@ -4,26 +4,27 @@ RELEASES
...
+6. Determine the version number for the next version (eg, `v1.X.Y`)
+7. Create a new folder in the `server/models/migrations/` directory to hold the migration file with the format `v1.A.B-v1.X.Y` where `1.A.B` is the current version and `v1.X.Y` is the next version.
+8. Move the `server/models/migrations/next/migrate.sql` file to the directory created in the previous step.
+9. Create an empty `migrate.sql` file in `server/models/migrations/next/`.  You may insert a comment at the top of the file including the name of the new release version.
+10. Commit updated files (`git commit ...`)
+11. Make sure your personal GITHUB_TOKEN environment variable is defined
     (assuming you have permissions to update the main BHIMA repository.  [See Github instructions for this.](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens)
-11. Run `yarn release` and follow the options.
-12. Verify the release was created and that the binary <version>.tar.gz file
+12. Run `yarn release` and follow the directions.
```
Comment (jniles) [inline RELEASES.md]: 👍

**Review by jniles — APPROVED** [review]
Changes look good.  Is the only real change to include making sure that the remote repositories are up to date?

File: RELEASES.md (hunk 2 above)
Comment (jmcameron) [reply RELEASES.md]: > 👍

Why delete the line to verify the release has the binary artifacts?

Comment (jmcameron) [reply RELEASES.md]: @jniles I think I forgot an update on the previous version RELEASES.md.  Please look over the this version.  Thanks.

**Comment by jniles** [comment]
This looks good to me.

**Pull request merged**